keyform/management/commands/migrate_old_data.py

```diff
@@ -0,0 +1,107 @@
+import sqlite3
+from datetime import datetime
+from django.core.management.base import BaseCommand
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from django.utils.timezone import make_aware, utc
+from keyform.models import *
+
+
+class Command(BaseCommand):
+
+    def handle(self, *args, **kwargs):
+        with transaction.atomic():
+            c = sqlite3.connect('/tmp/keyform.db').cursor()
+
+            buildings = {}
+            for row in c.execute('select building_id, building_name from building'):
+                buildings[str(row[0])] = Building.objects.get_or_create(name=row[1])[0]
+
+            statuses = {
+                '1': Status.objects.get_or_create(name='Processing', order=1)[0],
+                '2': Status.objects.get_or_create(name='Key Sent', order=2)[0],
+                '3': Status.objects.get_or_create(name='Key Distributed', order=3)[0],
+            }
+            reasons = {'1': 'dk', '2': 'lk', '3': 'sk'}
+            payment_methods = {'1': None, '2': 'ca', '3': 'ch'}
+            key_types = {'1': 'rm', '2': 'mb', '3': 'ot'}
+
+            old_contact_types = {
+                'Primary': ['Blair-Shannon House', 'Freudenberger House', 'Hammons House', 'Hutchens House', 'Kentwood Hall', 'Scholars House', 'Sunvilla Tower', 'Wells House', 'Woods House', 'Monroe'],
+                'Billing': ['Blair-Shannon House', 'Freudenberger House', 'Hammons House', 'Hutchens House', 'Kentwood Hall', 'Scholars House', 'Sunvilla Tower', 'Wells House', 'Woods House', 'Monroe'],
+                'Receptionist_Blair-Shannon': ['Blair-Shannon House'],
+                'Receptionist_Dogwood': [],
+                'Receptionist_Freudenberger': ['Freudenberger House'],
+                'Receptionist_Hammons': ['Hammons House'],
+                'Receptionist_Hutchens': ['Hutchens House'],
+                'Receptionist_Kentwood': ['Kentwood Hall'],
+                'Receptionist_Scholars': ['Scholars House'],
+                'Receptionist_Sunvilla': ['Sunvilla Tower'],
+                'Receptionist_Wells': ['Wells House'],
+                'Receptionist_Woods': ['Woods House'],
+                'Receptionist_Monroe': ['Monroe'],
+            }
+
+            contact_types = {}
+            for row in c.execute('select contact_type_id, contact_type_name from contact_type'):
+                contact_types[str(row[0])] = [Building.objects.get(name=x) for x in old_contact_types[row[1]]]
```

Comment:
It can be. I kinda wanted it to raise an Exception if there's a contact for a building that doesn't exist, though.